docs/snippets/angular/component-story-static-asset-with-import.ts.mdx

```diff
@@ -1,24 +1,25 @@
 ```ts
 // MyComponent.stories.ts
 
-import { Meta } from '@storybook/angular/types-6-0';
+import { MyComponent } from './MyComponent.component';
 
 import imageFile from './static/image.png';
 
 export default {
-  title: 'img',
-} as Meta;
+  component: MyComponent,
+};
 
 const image = {
   src: imageFile,
   alt: 'my image',
 };
 
-export const withAnImage = () => ({
-  template: `<img src="{{src}}" alt="{{alt}}" />`,
-  props: {
-    src: image.src,
-    alt: image.alt,
-  },
-});
+export const withAnImage = {
```

Comment:
```suggestion
export const WithAnImage = {
```

For consistency?

Reply:
Sounds good to me. I'm updating the snippets accordingly.

Reply:
This example is unified with the latest commit. Marking this as solved.